SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -257,6 +282,17 @@ private static int CreateSimDeployments(string[] args, bool useChinaPlatform)
                     return 1;
                 }
 
+                Console.WriteLine("Successfully created the simulated player deployment");
+
+                // Update coordinator worker flag for simulated player deployment to notify target deployment is ready.
+                simPlayerDeployment.WorkerFlags.Add(new WorkerFlag
+                {
+                    Key = "target_deployment_ready",
+                    Value = autoConnect.ToString(),
+                    WorkerType = CoordinatorWorkerName
+                });
+                deploymentServiceClient.UpdateDeployment(new UpdateDeploymentRequest { Deployment = simPlayerDeployment });
+
                 Console.WriteLine("Done! Simulated players will start to connect to your deployment");
```

Comment:
Shouldn't this log be different depending on whether autoconnect is on or off?

Reply:
True